contrib/go-redis/redis/redis.go

```diff
@@ -133,6 +148,16 @@ func commandsToString(cmds []redis.Cmder) string {
 	return b.String()
 }
 
+// Pipelined executes a function parameter to build a Pipeline and then immediately executes the built pipeline.
+func (c *Pipeliner) Pipelined(fn func(redis.Pipeliner) error) ([]redis.Cmder, error) {
+	if err := fn(c); err != nil {
+		return nil, err
+	}
+	cmds, err := c.Exec()
+	_ = c.Close()
```

Comment:
Why are we ignoring this error?

Reply:
`Close` will actually never return an error: https://github.com/go-redis/redis/blob/master/pipeline.go#L71

I would guess they wanted it to conform to the `Closer` interface in the `io` standard library.